kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -63,6 +65,9 @@
   public static final String REGION_CONFIG = "s3.region";
   public static final String REGION_DEFAULT = Regions.DEFAULT_REGION.getName();
 
+  public static final String ACL_CANNED_CONFIG = "s3.acl.canned";
+  public static final String ACL_CANNED_DEFAULT = null;
```

Comment:
After implementing the map of header values to ACLs, the change to null for default actually became straight-forward, so I committed that change.